src/utils/format_result.ts

```diff
@@ -3,14 +3,18 @@ import { RuntimeName } from '../runtimes/runtime'
 
 import { removeUndefined } from './remove_undefined'
 
-type FunctionResult = Omit<FunctionArchive, 'runtime'> & { runtime: RuntimeName; schedule?: string }
+type FunctionResult = Omit<FunctionArchive, 'runtime'> & {
+  runtime: RuntimeName
+  schedule?: string
+}
 
 // Takes the result of zipping a function and formats it for output.
 const formatZipResult = (archive: FunctionArchive) => {
   const functionResult: FunctionResult = {
     ...archive,
+    inSourceConfig: undefined,
```

Comment:
why not this?

```suggestion
    inSourceConfig: archive.inSourceConfig,
```

Reply:
Because we want the individual properties generated by ISC (e.g. `schedules`) to be part of the result, not the `inSourceConfig` property itself.

Reply:
But why do we set it to undefined then, instead of not including it at all?

Reply:
Because `inSourceConfig` is part of `archive` but we don't want it to be part of `functionResult`, so we must set it to `undefined` so that `removeUndefined` will get rid of it.